src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,428 @@
+# Microscopy
+
+Support for Microscopy was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+<!---
+The following link is temporary and points to the bep031 branch of bids-examples,
+update to 'https://github.com/bids-standard/bids-examples' when examples are merged.
+-->
+Microscopy datasets formatted using this specification are available on the
+[BIDS examples repository](https://github.com/neuropoly/bids-examples/tree/bep031)
+(`microscopy_SEM001` and `microscopy_SPIM001`) and can be used for practical
+guidance when curating a new dataset.
+
+Further Microscopy datasets are available:
+
+-   In PNG format:  [`data_axondeepseg_sem`](https://doi.org/10.5281/zenodo.5498378)
+-   In OME-TIFF format: [`Broca's Area Light-Sheet Microscopy`](https://doi.org/10.5281/zenodo.5517223)
```

Comment:
I wonder why bother with `'ing here since it not some "code" (name)?

```suggestion
-   In OME-TIFF format: [Broca's Area Light-Sheet Microscopy](https://doi.org/10.5281/zenodo.5517223)
```

Reply:
This was for consistency with other modalities (ex: https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/03-electroencephalography.html). It also seems to be consistent with recent changes to examples in PR #845.